test/filters/browser_tests.filter

```diff
@@ -5,6 +5,76 @@
 ## why the filter is required and create an associated tracking issue if needed.
 ##
 
+# Chromium ads are disabled
+-AdTaggingBrowserTest.*
+-AdClickLandingPageMetricsBrowserTest.*
+-AdTaggingFencedFrameBrowserTest.*
+-All/AutomaticLazyLoadFrameBrowserTest.*
+-LargeStickyAdViolationBrowserTest.*
+-LargeStickyAdViolationBrowserTestWithoutEnforcement.*
+-OverlayPopupAdViolationBrowserTest.*
+-OverlayPopupAdViolationBrowserTestWithoutEnforcement.*
+-Browser/SubresourceFilterDnsAliasFilteringThrottleBrowserTest.*
+-Renderer/SubresourceFilterDnsAliasFilteringThrottleBrowserTest.*
+-Renderer/SubresourceFilterDnsAliasResourceLoaderBrowserTest.*
+
+# Probably failing because of feature changes - need to investigate
+-WithOverrideParams/ReferrerOverrideTest.CrossOriginDowngradeNavigation/ParamCapReferrerToOriginOnCrossOrigin
+-NotificationPermissionBrowserTest.NonPersistentNotificationThirdPartyCountMetricTest # Histogram does not exist
+-RuntimeGetContextsApiTest.GetOffscreenDocumentContext
+-OpenWindowFromNTPBrowserTest.TransferFromNTPCreateNewTab # not InstantProcess
+-PasswordDialogViewTest.PopupAccountChooserWithMultipleCredentialsReturnEmpty
+-NewTabPageNavigationThrottleFencedFrameTest.FencedFrameDoesNotResetNewTabStartTime
+-CrossOriginRedirectReduceAcceptLanguageOTBrowserTest.*
+-DisableFeatureReduceAcceptLanguageBrowserTest.*
+-DisableReduceAcceptLanguageOTBrowserTest.*
+-SameOriginRedirectReduceAcceptLanguageOTBrowserTest.*
+-SerialBlocklistTest.*
+-SerialTest.*
+-ThirdPartyReduceAcceptLanguageOTBrowserTest.*
+-WebBluetoothPolicyTest.*
+-WebBluetoothTest.*
+-ChromeWebUsbTest.*
+-All/BrowserFrameColorProviderTest.*
+-All/WebBluetoothPermissionsPolicyTest.*
+-ContextMenuFencedFrameTest.AutomaticBeaconSentAfterContextMenuNavigation
+-CookiesTreeModelBrowserTest.NonDefaultStoragePartition
+-AlternativeErrorPageOverrideInfoBrowserTest.*
+-All/PrefersColorSchemeTest.*
+-AudioLifetimeEnforcerBrowserTest.DocumentActiveWhilePlayingAudio
+-AudioLifetimeEnforcerBrowserTest.DocumentInactiveIfNeverPlayedAudio
+-OffscreenDocumentBrowserTest.*
+-OffscreenDocumentManagerBrowserTest.*
+-SameOriginReduceAcceptLanguageOTBrowserTest.*
+-WebViewUsbTest.*
+-ChromeWebPlatformSecurityMetricsBrowserTest.*
+-OffscreenApiTest.*
+-WebAppFileHandlingBrowserTest.*
+-IsolatedOriginNTPBrowserTest.IsolatedOriginDoesNotInterfereWithNTP
+-CredentialManagerAvatarTest.AvatarFetchIsolatedPerOrigin
+-ProfilePickerViewBrowserTest.ShowScreen_FinishesForErrorOnStandardNavigation
+
+# Probably shouldn't be failing - need to investigate
```

Comment:
i'd file a ticket and put  a link there, otherwise these are going to be forever forgotten

Reply:
yep, I plan to create a follow-up issue for this

Reply:
https://github.com/brave/brave-browser/issues/34185